.github/workflows/build.yaml

```diff
@@ -50,7 +55,10 @@ jobs:
         chmod 700 ~/.ssh
         chmod 600 ~/.ssh/buildrunner-deploy-*
     - name: Test with pytest
-      run: pytest -v --junitxml=test-reports/test-results.xml
+      run: |
+        pytest -v -m "not serial" --numprocesses=auto --junitxml=test-reports/serial-test-results.xml
```

Comment:
Shouldn't these file names be switched?

Reply:
Yes. Only if you don't want to confuse people.

Reply:
Fixed